flytestdlib/storage/storage.go

```diff
@@ -40,6 +40,40 @@ type Metadata interface {
 	ContentMD5() string
 }
 
+type CursorPosition int
+
+const (
+	StartCursorPos  CursorPosition = 0
```

Comment:
Could we add some comments to these numbers and to the struct please?  Also can we rename the constants?  These aren't really cursor positions right?  These are more like cursor types.

Reply:
will do